python/capypdf.py

```diff
@@ -155,6 +155,7 @@ class OptionalContentGroupId(ctypes.Structure):
     ctypes.c_double, ctypes.c_double, ctypes.c_double, ctypes.c_double, ctypes.c_double, ctypes.c_double]),
 ('capy_dc_cmd_cm', [ctypes.c_void_p,
     ctypes.c_double, ctypes.c_double, ctypes.c_double, ctypes.c_double, ctypes.c_double, ctypes.c_double]),
+('capy_dc_cmd_d', [ctypes.c_void_p, POINTER(ctypes.c_double), ctypes.c_int32, ctypes.c_double]),
```

Comment:
This should not work. The third argument is an `int32` but the C API has a `size_t`, which is 64 bits on most platforms nowadays.

Reply:
Fixed